src/components/Home.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import styles from '../CSS/Home.module.css';
+
+const Home = () => {
+  const { news, categories, totalResults } = useSelector((state) => state.news);
+  const [selectedItem, setSelectedItem] = useState(null);
+
+  const handleCardClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className={styles.homeCont} data-testid="home">
+      <div className={styles.headCont}>
+        <h1 className={styles.titleN}>News</h1>
+        <p>
+          Total Results Sum:
+          {' '}
+          {totalResults}
+        </p>
+      </div>
+      <div className={styles.homeCard}>
+        {news.map((item) => (
+          <div className={styles.card} key={item.title} data-testid="card">
+            <Link to={`/detail/${encodeURIComponent(item.title)}`} className={styles.single_card}>
+              <div
+                onClick={() => handleCardClick(item)}
+                onKeyDown={() => handleCardClick(item)}
```

Comment:
- [ ] As part of the requirements, The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.  kindly add the functionalities of filtering since your app missing it 🙏